src/wstool/multiproject_cmd.py

```diff
@@ -579,7 +579,7 @@ def do_work(self):
     managed_paths = [os.path.join(path, e.get_local_name()) for e in elements]
     unmanaged_paths = []
     scm_clients = {SvnClient: 'svn', GitClient: 'git', BzrClient:'bzr', HgClient:'hg'}
-    for root, dirs, files in os.walk(path):
+    for root, dirs, files in os.walk(path, followlinks=True):
```

Comment:
I wonder if it would be possible to add a test to show this new feature. There are already some good tests in this repo.

Reply:
I don't have resources to add a test.
By the way, what would you like to test here? `os.walk` is a system function, that should be tested upstream.